src/types.ts

```diff
@@ -249,6 +250,7 @@ export interface ValueChangedEventConfig {
 
 export interface TxStateChangedEventConfig {
   tx_hash: string;
+  timeout: number;
```

Comment:
could you add 'unit' in the variable name, e.g. timeout_ms?

Reply:
Fixed. The blockchain code will also be modified. Thanks!